File: app/src/main/java/com/motrixi/datacollection/network/models/DataInfo.java
package com.motrixi.datacollection.network.models;

import java.io.Serializable;

/**
 * author : Jason
 * date   : 2020/12/8 10:50 AM
 * desc   :
 */
 public class DataInfo implements Serializable {

     public String advertisingId;
     public String appKey;
     public String email;
     public String imei;
     public String operationSystem;
     public String language;
     public String installedApplication;
     public String location;
     public String userAgent;
     public String deviceMake;
     public String deviceModel;
     public String ipAddress;
     public String mcc;
     public String deviceID;
     public String serial;
     public String androidID;
     public String consentFormID;


    @Override
    public String toString() {
        return "DataInfo{" +
                "advertisingId='" + advertisingId + '\'' +
                ", appKey='" + appKey + '\'' +
                ", email='" + email + '\'' +
                ", imei='" + imei + '\'' +
                ", operationSystem='" + operationSystem + '\'' +
                ", language='" + language + '\'' +
                ", installedApplication='" + installedApplication + '\'' +
                ", location='" + location + '\'' +
                ", userAgent='" + userAgent + '\'' +
                ", deviceMake='" + deviceMake + '\'' +
                ", deviceModel='" + deviceModel + '\'' +
                ", ipAddress='" + ipAddress + '\'' +
                ", mcc='" + mcc + '\'' +
                ", deviceID='" + deviceID + '\'' +
                ", serial='" + serial + '\'' +
                ", androidID='" + androidID + '\'' +
                ", consentFormID='" + consentFormID + '\'' +
                '}';
    }
}

File: appsdk/src/main/java/com/motrixi/datacollection/network/models/DataInfo.kt
package com.motrixi.datacollection.network.models

import java.io.Serializable

/**
 * @author jsqi
 * @time 2019/12/4 14:54
 */
class DataInfo: Serializable {

    var advertisingId: String? = null
    var appKey: String? = null
    var email: String? = null
    var imei: String? = null
    var operationSystem: String? = null
    var language: String? = null
    var installedApplication: String? = null
    var location: String? = null
    var userAgent: String? = null
    var deviceMake: String? = null
    var deviceModel: String? = null
    var ipAddress: String? = null
    var mcc: String? = null
    var deviceID: String? = null
    var serial: String? = null
    var androidID: String? = null
    var privacyConsent: Int? = 1
    override fun toString(): String {
        return "DataInfo(advertisingId=$advertisingId, appKey=$appKey, email=$email, imei=$imei, operationSystem=$operationSystem, language=$language, installedApplication=$installedApplication, location=$location, userAgent=$userAgent, deviceMake=$deviceMake, deviceModel=$deviceModel, ipAddress=$ipAddress, mcc=$mcc, deviceID=$deviceID, serial=$serial, androidID=$androidID, privacyConsent=$privacyConsent)"
    }


}